src/App.tsx

```diff
@@ -15,49 +16,109 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+export function getReorderedGoods(
+  goods: string[],
+  sortType: SortType,
+  isReversed: boolean,
+) {
+  const visibleGoods = [...goods];
+
+
+  visibleGoods.sort((good1: string, good2: string) => {
+    switch (sortType) {
+      case SortType.ALPHABET:
+
+        return good1.localeCompare(good2);
+      case SortType.LENGTH:
+
+        return good1.length - good2.length;
+
+      default:
+        return 0;
+    }
+  });
```

Comment:
better to use the switch first and then sort all the cases
and do not forget to handle all cases 
what about SortType.NONE ?